docs/content/releases.md

```diff
@@ -125,29 +125,31 @@ We will provide technical support for NGINX Ingress Controller on any Kubernetes
 
 <hr>
 
-## 3.2.1
+## NGINX Ingress Controller 3.2.1
```

Comment:
Do we want to also change the heading of release `3.3.0` `3.3.1` and `3.3.2`? Since they don't have any titles, it displays them differently on the right pane compared to every other release. Now maybe this is exactly what you were going for. Just wanted to be sure.

<img width="1551" alt="image" src="https://github.com/nginxinc/kubernetes-ingress/assets/100570330/143aabbe-4f68-4c8d-9446-9e975c2f5ce2">

Reply:
Got confirmation this is an error: pushing a change now to reflect the current main/release state with iconography.

Reply:
Move this to a separate Jira ticket. The RNs aren't in scope for this PR.